yarn-project/p2p/src/services/peer-manager/peer_manager.ts

```diff
@@ -205,6 +266,7 @@ export class PeerManager {
     const dialQueue = this.libP2PNode
       .getDialQueue()
       .filter(peer => !!peer.peerId)
+      .filter(peer => !this.isPrivatePeer(peer.peerId!))
```

Comment:
I dont think we need to exclude private peers from this query, It should not be a publically accessible method. 

More just exclude them from being broadcast in discovery